docs/source/concepts/FittingMinimizers.rst

```diff
@@ -0,0 +1,205 @@
+terms of how they perform when fitting several benchmark
+problems. This is a relative comparison in the sense that for every
+problem the best possible results with Mantid minimizers are given a
+top score of "1". The ranking is continuous and the score of a
+minimizer represents the ratio between its performance and the
+performance of the best. We compare accuracy and run time.
+
+.. Q: accuracy metric: sum of squared residuals or chi2 variants
+
+For example, a ranking of 1.25 for a minimizer for a given problem
+means:
+
+- Referring to the accuracy of a minimizer, it produces a solution
+  with squared residuals 25% larger than the best solution in Mantid.
+- Referring to the run time, it takes 25% more time than the fastest
+  minimizer.
+
+All the minimizers available in Mantid 3.7 were compared, with the
+exception of FABADA which belongs to a different class of methods and
+would not be compared in a fair manner. Minimizers are used here as
+"black boxes", without any special initialization, constraints, or
+rewriting of the equations to fit.
+
+Accuracy is measured using the sum of squared fitting errors as
+metric, or "ChiSquared" as defined in :ref:`Fit
+<algm-CalculateChiSquared>`, where the fitting errors are the
+difference between the expected outputs and the outputs calculated by
+the model fitted: :math:`\chi_{1}^{2} = \sum_{i} (y_i - f_i)^2` (see
+:ref:`CalculateChiSquared <algm-CalculateChiSquared>` for full details
+and different variants).  Run time is measured for the execution of
+the :ref:`Fit <algm-Fit>` algorithm for an equation and dataset
+previously created.
+
+.. There would be two alternative for the errors:
+   1. Without errors, as it is
+   2. With "simulated" sqrt(X) errors
+   Here we use 2 because Levenberg-Marquardt doesn't work with 1 ('Unweighted least squares' cost function
+
+The cost function used in all cases is `Least squares` (weighted by
+the errors, as usually applied in Mantid).
+
+
+Benchmark problems
+##################
+
+Each test problem included in this comparison is defined by the
+following information:
+
+- Input data (:math:`x_i` values)
+- Output data (:math:`y_i` values)
+- Function to fit, with parameters
+- Starting point or initial values of the function parameters
+- Certified or reference best values for the parameters, with an associated residual of the certified or best model 
+
+The problems have been obtained from two different benchmarks:
+
+- `NIST nonlinear regression problems <http://itl.nist.gov/div898/strd/general/dataarchive.html>`__.
+
+- The `Constrained and Unconstrained Testing Environment <http://www.cuter.rl.ac.uk/Problems/mastsif.html>`__
+  set of test problems for linear and nonlinear optimization.
+
+As the NIST problems do not include measurement errors, assuming that
+these datasets would represent spectra from a typical Mantid workspace
+we introduce errors as the square root of the observations.
```

Comment:
I agree that this is very arguable, and I'm myself uneasy with this, as it effectively means that we are not testing the real NIST  benchmark.
But the problem is that `Levenberg-Marquardt` doesn't behave if there are no errors. So that's the only alternative that I saw. Otherwise we would need to take out the best minimizer, or would there be an alternative?
On the other hand the comparison tables would look pretty similar.